host_os.py

```diff
@@ -32,11 +34,36 @@
     'build-iso': build_iso,
 }
 
+def is_package_installed(package_name):
+    """
+    Checks if a RPM package is installed
+    """
+
+    cmd = "rpm -q %s" % package_name
+    try:
+        run_command(cmd, shell=True)
+    except SubprocessError as e:
+        # grep returns 1 when search string is not found and 2 if an error occurred
+        if e.returncode == 1:
+            return False
+        else:
+            raise
+
+    return True
+
 
 if __name__ == '__main__':
     CONF = config.setup_default_config()
     subcommand = CONF.get('default').get('subcommand')
 
+    PACKAGE_DEPENDENCIES_FILE_PATH = "rpm_requirements.txt"
+    package_dependencies = open(PACKAGE_DEPENDENCIES_FILE_PATH).readlines()
+    package_dependencies = [x.strip('\n') for x in package_dependencies]
+    for package in package_dependencies:
+        if not is_package_installed(package):
+            print("%s package should be installed before running this script" % package)
+            sys.exit(4)
+
```

Comment:
MO> The PACKAGE_DEPENDENCIES_FILE_PATH is a huge variable name and it is used only once. I'd use a variable if its value comes from a configuration file. Since its value is hard-coded here, I don't see the point for a variable.

This is a constant. Although Python does not have the concept of constant, I always prefer to use them when hardcoding strings (eg files names/paths). It is usually a good programming practice and make code reading easier (imho). If other devs agree with you, I may remove the constant.

MO>  I guess it would be better for user to know all package names that there are missing on his system in one shot.

I agree. Done.

MO> What about defining an error code when packages are not installed, similar to what has been proposed in pull 143?

I'm already returning a specific error code (4) for this situation. I'll create a constant for it.

MO> I'd also use list-comprehension to simplify this block. 

This is a matter of taste. I personally like more the traditional for loop, but I know most developers of our team prefer the list comprehension, so I'll use it.

OH> just adding that readlines does not return line separators, so the .strip('\n') could be removed.

I don't think so.

>>> open("bla.txt").readlines()
['a\n', 'b\n', 'c\n', 'd\n']